src/load/HISTORY.rst

```diff
@@ -4,6 +4,11 @@ Release History
 ===============
 
 
+1.5.1
+++++++
+* Add support for engine reference identity using CLI. Engine reference identity can be set using `--engine-ref-id-type` and `--engine-ref-ids` argument in 'az load test create' and 'az load test update' commands. Engine reference identity set in YAML config file under key `referenceIdentities` with `kind` as `Engine` will also be honoured.
+
```

Comment:
Adding params is taken as feature for cli extension modules and asks a minor version upgrade, therefore the new version should be `1.6.0` instead of `1.5.1`.

For more info about cli extension versioning schema, please refer to: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md